src/components/forms/attendee-form/attendee-form.js

```diff
@@ -283,7 +296,7 @@ class AttendeeForm extends React.Component {
 
                 <div className="row">
                     <div className="col-md-12 submit-buttons">
-                        <input type="button" onClick={this.handleSubmit}
+                        <input type="button" onClick={() => this.formRef.current.dispatchEvent(new Event("submit", { cancelable: true, bubbles: true }))}
```

Comment:
lets move this call to its on handler ( lets avoid the inline )
name it with a meaningful name like triggerFormSubmit or something else